uavcan/driver/python_can.py

```diff
@@ -0,0 +1,157 @@
+#
+# Copyright (C) 2019 UAVCAN Development Team  <uavcan.org>
+#
+# This software is distributed under the terms of the MIT License.
+#
+# Author: Jos Vos <jos.vos@demcon.nl>
+#
+
+from __future__ import division, absolute_import, print_function, unicode_literals
+import os
+import errno
+import time
+import threading
+from logging import getLogger
+
+from .common import DriverError, TxQueueFullError, CANFrame, AbstractDriver
+from .timestamp_estimator import TimestampEstimator
+
+try:
+    import queue
+except ImportError:
+    # noinspection PyPep8Naming,PyUnresolvedReferences
+    import Queue as queue
+
+logger = getLogger(__name__)
+
+try:
+    import can
+except ImportError:
+    logger.error("no module 'can' available, please install python-can")

```

Comment:
This can't be an error. If the user doesn't care about Python-CAN, there is no use warning them about a library they don't even need being unavailable. I suggest demoting this to INFO.

Reply:
Agreed, good idea.